providers/shared/flows/context/context.ftl

```diff
@@ -781,7 +900,7 @@ The roles available are defined as part of the state of the target component.
             link.Id!"COTFatal: Id not provided on link",
             link.Name!"COTFatal: Name not provided on link",
             removeObjectAttributes(link, ["Id", "Name", "Direction", "Role", "Attributes", "Enabled", "Type"]),
-            (link.Direction?lower_case)!OUTBOUND_LINK_DIRECTION,
+            link.Direction!OUTBOUND_LINK_DIRECTION,
```

Comment:
need to pick up this change from previously merged PR

Reply:
Fixed, thought I had caught all of that